examples/6-advanced-flows.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Run Scripts from chat-bubble</title>
+
+	<!-- for mobile screens -->
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+	<!-- stylesheets are conveniently separated into components -->
+	<link rel="stylesheet" media="all" href="../component/styles/setup.css">
+	<link rel="stylesheet" media="all" href="../component/styles/says.css">
+	<link rel="stylesheet" media="all" href="../component/styles/reply.css">
+	<link rel="stylesheet" media="all" href="../component/styles/typing.css">
+	<link rel="stylesheet" media="all" href="../component/styles/input.css">
+	<style>
+	body {
+		background: #dcdde0;
+	}
+	.bubble-container {
+		height: 100vh;
+	}
+	.bubble-container .input-wrap textarea {
+		margin: 0;
+		width: calc(100% - 30px);
+	}
+	</style>
+</head>
+<body>
+
+<!-- container element for chat window -->
+<div id="chat"></div>
+
+<!-- import the JavaScript file -->
+<script src="../component/Bubbles.js"></script>
+<script>
+    // initialize by constructing a named function...
+    var chatWindow = new Bubbles(
+      document.getElementById("chat"), // ...passing HTML container element...
+"chatWindow" // ...and name of the function as a parameter
+    )
+    // `.talk()` will get your bot to begin the conversation
+    chatWindow.talk(
+      // pass your JSON/JavaScript object to `.talk()` function where
+// you define how the conversation between the bot and user will go
+{
+    // "ice" (as in "breaking the ice") is a required conversation object
+    // that maps the first thing the bot will say to the user
+        ice: {
+        // "says" defines an array of sequential bubbles
+        // that the bot will produce
+        says: [
+            "Hello",
+            "We're exploring chat flows.",
+            "<img src=https://i0.wp.com/meanbusiness.com/wp-content/uploads/2018/03/Coral-Share_Ideas-350x198.gif?fit=300 />",
+            "Do you have 90 seconds to spend with us?",
+            "(You can win Prizes!) [Sidetrack to the main flow]",
+        ],
+        // "reply" is an array of possible options the user can pick from
+        // as a reply
+      reply: [
+            {
+        question: "Sure", // label for the reply option
+    answer: "chapter-one" // key for the next conversation object
+            },
+            {
+        question: "No. Thanks [Escape Valve hit]", // label for the reply option
```

Comment:
Not sure if the information in the square brackets is helpful in this chat. Perhaps we can explain in plain English what the bot is doing?

Reply:
Understood. As you note above, the majority of people will experience the example front-end first, rather than via code comments. I'll update accordingly.